definitions/json/CaseTypeTab/CaseTypeTab.json

```diff
@@ -928,160 +928,229 @@
     "Channel": "CaseWorker",
     "TabID": "initialConsideration",
     "TabLabel": "Initial Consideration",
+    "TabDisplayOrder": 7,
     "CaseFieldID": "initialConsiderationTabTitle",
-    "TabFieldDisplayOrder": 1,
-    "TabDisplayOrder": 7
+    "TabFieldDisplayOrder": 1
   },
   {
     "CaseTypeID": "ET_EnglandWales",
     "Channel": "CaseWorker",
     "TabID": "initialConsideration",
     "TabLabel": "Initial Consideration",
-    "TabDisplayOrder": 7,
     "CaseFieldID": "etInitialConsiderationDocument",
     "TabFieldDisplayOrder": 2
   },
   {
     "CaseTypeID": "ET_EnglandWales",
     "Channel": "CaseWorker",
     "TabID": "initialConsideration",
-    "CaseFieldID": "etICJuridictionCodesInvalid",
+    "CaseFieldID": "icReceiptET3FormIssues",
     "TabFieldDisplayOrder": 3
   },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "Channel": "CaseWorker",
+    "TabID": "initialConsideration",
+    "CaseFieldID": "icRespondentsNameIdentityIssues",
+    "TabFieldDisplayOrder": 4
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "Channel": "CaseWorker",
+    "TabID": "initialConsideration",
+    "CaseFieldID": "icJurisdictionCodeIssues",
+    "TabFieldDisplayOrder": 5
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "Channel": "CaseWorker",
+    "TabID": "initialConsideration",
+    "CaseFieldID": "icApplicationIssues",
+    "TabFieldDisplayOrder": 6
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "Channel": "CaseWorker",
+    "TabID": "initialConsideration",
+    "CaseFieldID": "icEmployersContractClaimIssues",
+    "TabFieldDisplayOrder": 7
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "Channel": "CaseWorker",
+    "TabID": "initialConsideration",
+    "CaseFieldID": "icClaimProspectIssues",
+    "TabFieldDisplayOrder": 8
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "Channel": "CaseWorker",
+    "TabID": "initialConsideration",
+    "CaseFieldID": "icListingIssues",
+    "TabFieldDisplayOrder": 9
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "Channel": "CaseWorker",
+    "TabID": "initialConsideration",
+    "CaseFieldID": "icDdaDisabilityIssues",
+    "TabFieldDisplayOrder": 10
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "Channel": "CaseWorker",
+    "TabID": "initialConsideration",
+    "CaseFieldID": "icOrderForFurtherInformation",
+    "TabFieldDisplayOrder": 11
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "Channel": "CaseWorker",
+    "TabID": "initialConsideration",
+    "CaseFieldID": "icOtherIssuesOrFinalOrders",
+    "TabFieldDisplayOrder": 11
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "Channel": "CaseWorker",
+    "TabID": "initialConsideration",
+    "CaseFieldID": "icCompletedBy",
+    "TabFieldDisplayOrder": 12
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "Channel": "CaseWorker",
+    "TabID": "initialConsideration",
+    "CaseFieldID": "icDateCompleted",
+    "TabFieldDisplayOrder": 13
+  },
```

Comment:
these should be the last 2 fields on the tab

Reply:
I have put etICJuridictionCodesInvalid & etICInvalidDetails to the bottom of the list

Reply:
the date and who it was completed by should be the last 2 things on the tab

Reply:
Ah sorry, I picked the wrong two. I have moved data and who to the bottom of the list